messaging/nats.go

```diff
@@ -1,20 +1,23 @@
 package messaging
 
 import (
+	"fmt"
 	"strings"
 
 	"github.com/nats-io/nats"
 	"github.com/sirupsen/logrus"
 
 	"github.com/rybit/nats_logrus_hook"
 
+	"github.com/netlify/netlify-commons/discovery"
 	"github.com/netlify/netlify-commons/tls"
 )
 
 type NatsConfig struct {
-	TLS         *tls.Config `mapstructure:"tls_conf"`
-	Servers     []string    `mapstructure:"servers"`
-	LogsSubject string      `mapstructure:"log_subject"`
+	TLS            *tls.Config `mapstructure:"tls_conf"`
+	ServiceDNSName string      `mapstructure:"service_dnsname"`
```

Comment:
can you add an `envconfig:"dns_name"` so that we can access it by envvars easily? It is redundant to say "service_" b/c it is embedded in a nats config already

Reply:
I wanted to add "service" to make it explicit that it was a "discovery" DNS name, not a simple DNS name pointing to a host